source/local/local.go

```diff
@@ -193,9 +212,6 @@ func (ls *localSourceHandler) snapshot(ctx context.Context, s session.Group, cal
 	}
 
 	if err := filesync.FSSync(ctx, caller, opt); err != nil {
-		if status.Code(err) == codes.NotFound {
```

Comment:
Did you understand why this did not work?

Reply:
If you are referring to our issue, no we haven't root caused it yet.